src/crispy.h

```diff
@@ -33,6 +33,9 @@
 #define BETWEEN(l,u,x) (((l)>(x))?(l):((x)>(u))?(u):(x))
 #endif
 
+#define CRISPY_FPSLIMIT_MIN 35
+#define CRISPY_FPSLIMIT_MAX 500
```

Comment:
What's the actual limiting factor for fps in Crispy? Timer precision? Just wondering.

Reply:
Yes, I believe it’s the millisecond timer precision. I set this based on quick and dirty testing on my system. Using the showfps cheat the frame limiter started getting inaccurate with a limit set slightly above 600 Hz.